addons/website/views/snippets/snippets.xml

```diff
@@ -490,6 +490,7 @@
             <t t-snippet="website.s_icon" string="Icon" t-thumbnail="/website/static/src/img/snippets_thumbs/s_icon.svg"/>
             <t t-snippet="website.s_accordion" string="Accordion" t-thumbnail="/website/static/src/img/snippets_thumbs/s_accordion.svg"/>
             <t t-snippet="website.s_alert" string="Alert" t-thumbnail="/website/static/src/img/snippets_thumbs/s_alert.svg"/>
+            <t t-snippet="website.s_list_group" string="List Group" t-thumbnail="/website/static/src/img/snippets_thumbs/s_list_group.svg"/>
```

Comment:
I find it a bit too big when dropped as a grid item, maybe you should add `t-grid-column-span="4"` (or 5 because of the text wrapping when the container is small, but I don't really find it annoying so 4 is fine) :slightly_smiling_face: